code/game/machinery/doors/firedoor.dm

```diff
@@ -498,7 +498,7 @@
 /obj/machinery/door/firedoor/try_to_activate_door(mob/user, access_bypass = FALSE)
 	return
 
-/obj/machinery/door/firedoor/try_to_weld(obj/item/weldingtool/W, mob/user)
+/obj/machinery/door/firedoor/try_to_weld_secondary(obj/item/weldingtool/W, mob/user)
```

Comment:
Is this an unintentional change? I don't see it mentioned in the PR body

Reply:
this isnt included in the pr

Reply:
No clue why the firelock stuff is there, I think I accidentally comitted it from another pr I made a while ago. I’ll see if I can remove it

Reply:
Yeah I don’t know if I can remove the commit right now but I can try later when I’m at my PC